cpp/build.sh

```diff
@@ -394,10 +370,9 @@ cd $BASE_DIR
 git submodule update --init --recursive
 
 install_folly
-install_kineto
+#install_kineto
```

Comment:
Can you check if we actually need this for mac install? And lets remove it if its not necessary